main.tf

```diff
@@ -90,7 +90,7 @@ resource "aws_elasticache_subnet_group" "default" {
 
 resource "aws_elasticache_parameter_group" "default" {
   count       = module.this.enabled ? 1 : 0
-  name        = module.this.id
+  name        = var.parameter_group_name != null ? var.parameter_group_name : "${module.this.id}-${var.family}"
```

Comment:
Thank you @aknysh for code review!

I can change it, no problem. But first, I would like to ask: what is the advantage of creating a new local variable in this case? We will have a new local variable that will only be used once. Additionally, one line below, we define `description` without using the local variable.

If local variable is strongly preferred here then let me know to which `locals {}` block I should add it (because we have to) 

```
#
# Security Group Resources
#
locals {
  enabled              = module.this.enabled
...
}

...
locals {
  elasticache_subnet_group_name = var.elasticache_subnet_group_name != "" ? var.elasticache_subnet_group_name : join("", aws_elasticache_subnet_group.default[*].name)
  ...
}
```

Reply:
@aknysh done